Assets/HoloToolkit/Utilities/Scripts/HeadsUpDirectionIndicator.cs

```diff
@@ -100,16 +101,17 @@ private void Update()
             }
             else
             {
+                Camera mainCamera = CameraCache.main;
```

Comment:
IMO calling it directly actually helps legibility, even if it's just a bit longer. Less Spaghetti. 
Not that performance is an issue here, but local references do add to the heap (albeit negligible now days).  Not a big deal.